independent-projects/tools/codestarts/src/test/java/io/quarkus/devtools/codestarts/core/CodestartDataTest.java

```diff
@@ -1,7 +1,6 @@
 package io.quarkus.devtools.codestarts.core;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.*;
```

Comment:
wildcard * but why only this:

<img width="526" alt="image" src="https://github.com/user-attachments/assets/97529ae6-cfaa-4c36-8874-cf35a8d1a1f9" />

Reply:
- https://github.com/diffplug/spotless/issues/649